commit 37c962d195005d009e130e65a9e55960996c3cab
Author: Roland Vossen <rvossen@broadcom.com>
Date:   Wed Jun 29 16:48:22 2011 -0700

    staging: brcm80211: fix for reported log spam problem
    
    Every few minutes, this message would appear in syslog:
    
    ieee80211 ph0: wl_ops_bss_info_changed: BSS idle: true (implement)
    
    The message has been deleted, the driver requires no special action on this
    particular event (). See: https://bugzilla.kernel.org/show_bug.cgi?id=38162
    
    Reported-by: David Hill <hilld@binarystorm.net>
    Signed-off-by: Roland Vossen <rvossen@broadcom.com>
    Reviewed-by: Arend van Spriel <arend@broadcom.com>
    Reviewed-by: Franky Lin <frankyl@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmsmac/wl_mac80211.c
+++ b/drivers/staging/brcm80211/brcmsmac/wl_mac80211.c
@@ -449,11 +449,6 @@ wl_ops_bss_info_changed(struct ieee80211
 		wiphy_err(wiphy, "%s: qos enabled: %s (implement)\n", __func__,
 			  info->qos ? "true" : "false");
 	}
-	if (changed & BSS_CHANGED_IDLE) {
-		/* Idle changed for this BSS/interface */
-		wiphy_err(wiphy, "%s: BSS idle: %s (implement)\n", __func__,
-			  info->idle ? "true" : "false");
-	}
 	return;
 }
 
